parts/site-header.html

```diff
@@ -1,8 +1,13 @@
-<!-- wp:group {"layout":{"type":"constrained"}} -->
-<div class="wp-block-group"><!-- wp:group {"align":"wide","layout":{"type":"flex","justifyContent":"space-between"}} -->
-	<div class="wp-block-group alignwide"><!-- wp:site-title {"level":0} /--></div>
-	<!-- /wp:group -->
-</div>
+<!-- wp:group {"backgroundColor":"ucsc-primary-blue","layout":{"type":"constrained"}} -->
+<div class="wp-block-group has-ucsc-primary-blue-background-color has-background"><!-- wp:columns -->
+	<div class="wp-block-columns"><!-- wp:column {"style":{"spacing":{"padding":{"right":"0"}}}} -->
+		<div class="wp-block-column"><!-- wp:site-title {"style":{"elements":{"link":{"color":{"text":"var:preset|color|white"}}},"typography":{"textTransform":"uppercase"}},"textColor":"white"} /--></div>
```

Comment:
I think this will work and we'll certainly need to do some training for them on this. One thing I am noticing is that when I tested this locally, SVG support is not currently available. Can you confirm this and add support for SVGs to the project @MlKilderkin ?

Reply:
@Vinsanity should be fine now

Reply:
Thanks for adding that!